skymp5-server/cpp/server_guest_lib/PapyrusActor.cpp

```diff
@@ -76,3 +78,17 @@ VarValue PapyrusActor::SetAlpha(VarValue self,
   }
   return VarValue::None();
 }
+
+VarValue PapyrusActor::EquipItem(VarValue self,
+                                 const std::vector<VarValue>& arguments)
+{
+  if (auto actor = GetFormPtr<MpActor>(self)) {
+    auto& form = GetRecordPtr(arguments[0]);
+    if (!form.rec) {
+      throw std::runtime_error(
+        fmt::format("Unable to find a record of a form"));
+    }
+    actor->OnEquip(form.rec->GetId());
```

Comment:
You'll need something like.
```suggestion
    if (arguments.size() < 1) {
      throw std::runtime_error("EquipItem requires at least one argument");
    }
    SpSnippet(GetName(), "EquipItem", SpSnippetFunctionGen::SerializeArguments(arguments),  actor->->GetFormId()).Execute(actor);
```